lib/Controller/ContactIntegrationController.php

```diff
@@ -75,8 +80,13 @@ public function newContact(?string $contactName = null, ?string $mail = null): J
 	 * @return JSONResponse
 	 */
 	#[TrapError]
-	public function autoComplete(string $term): JSONResponse {
-		$res = $this->service->autoComplete($term);
-		return (new JSONResponse($res))->cacheFor(60 * 60, false, true);
+	public function autoComplete(string $term, bool $forceSAB = false): JSONResponse {
+		$cached = $this->cache->get($term);
```

Comment:
the cache key is not scoped to a user. you will cache-spill results from one user to another